controllers/players.ts

```diff
@@ -28,7 +29,25 @@ const handleAuth = async (req: Request, res: Response): Promise<void> => {
     res.status(201).send(payload);
 };
 
+const checkTeamExists = async (teamId: number): Promise<boolean> => {
+    const team = await Team.getTeamById(teamId);
+    if (team) return true;
```

Comment:
Just interested in how TS works: could you return team here and have it be a boolean?

Reply:
I think Google says !!team would work